docs/root/configuration/best_practices/edge.rst

```diff
@@ -1,5 +1,91 @@
+.. _best_practices_edge:
```

Comment:
Can you also add a FAQ entry entitled: "How do I configure Envoy as a secure edge proxy?" (or similar) and link to here?

Reply:
Added.